### PR TITLE
feat(slang) : added af

### PR DESCRIPTION
**Pull request by nsijwali**
Closes #

### What new changes did you make? Tick all applicable boxes
- ✅[ Added new abbreviation
- [ ] Fixed something in the source code
- [ ] Added a new feature
- [ ] Fixed the docs (README.md, CONTRIBTUING.md, etc)

### Describe the new changes you added.
image

### CODE REVIEW

File: public/server/db/af.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "definition": "As Fuck",
```
Comment (Njong392) [inline public/server/db/af.json:2]: This looks good, but I would remove the alternative meaning.

**Review by Njong392 — CHANGES_REQUESTED** [review]

File: public/server/db/af.json (hunk 1 above)
Comment (nsijwali) [reply public/server/db/af.json:2]: sure

**Review by nsijwali — COMMENTED** [review]
looks fine

**Comment by Njong392** [comment]
Hey @nsijwali do you need any help making updates to this pull request?

**Comment by nsijwali** [comment]
> Hey @nsijwali do you need any help making updates to this pull request?

done

**Review by Njong392 — APPROVED** [review]
This looks amazing! Thank you ⭐

**Pull request merged**